testkit/src/main/scala/org/bitcoins/eclair/rpc/EclairRpcTestUtil.scala

```diff
@@ -215,8 +226,100 @@ trait EclairRpcTestUtil extends BitcoinSLogger {
     ()
   }
 
+  private def createNodeLink(
+      bitcoindRpcClient: Option[BitcoindRpcClient],
+      channelAmount: MilliSatoshis)(implicit actorSystem: ActorSystem): (
+      EclairRpcClient,
+      EclairRpcClient,
+      EclairRpcClient,
+      EclairRpcClient) = {
+    implicit val ec: ExecutionContext = actorSystem.dispatcher
+    val internalBitcoind = bitcoindRpcClient.getOrElse(
+      BitcoindRpcTestUtil.startedBitcoindRpcClient()
+    )
+
+    val (first, second) = createNodePair(Some(internalBitcoind))
+    val (third, fourth) = createNodePair(Some(internalBitcoind))
+
+    def open(
+        c1: EclairRpcClient,
+        c2: EclairRpcClient): Future[FundedChannelId] = {
+      openChannel(c1,
+                  c2,
+                  channelAmount.toSatoshis,
+                  MilliSatoshis(channelAmount.toLong / 2))
+    }
+
+    EclairRpcTestUtil.connectLNNodes(second, third)
```

Comment:
nit: I would like to start using this syntax for things where we don't care about the return type

```scala

val _ = EclairRpcTestUtil.connectLNNodes(second, third)

```